scripts/check_frontend_test_coverage.py

```diff
@@ -57,6 +57,7 @@
     'question-update.service.ts',
     'rule-type-selector.directive.ts',
     'select2-dropdown.directive.ts',
+    'topic-editor-page.component.ts',
```

Comment:
I don't think this is OK -- have you gotten approval from @nithusha21? See https://github.com/oppia/oppia/blob/2dccb29305cbb855edf65919fd958701cf6a27e6/scripts/check_frontend_test_coverage.py#L38

Reply:
I agree. We should not be adding files to this list. Instead, this PR should include frontend tests to cover the changes to `topic-editor-page.component.ts`

Reply:
I will remove the file shortly. I added this because I was little stuck while writing the tests for this file.